lxc/storage_volume.go

```diff
@@ -17,7 +18,7 @@ import (
 	"github.com/spf13/cobra"
 	"go.yaml.in/yaml/v2"
 
```

Comment:
we dont import the client as `lxd` so please update your editor settings to not do this automatically